eclass/texlive-common.eclass

```diff
@@ -35,10 +33,10 @@ TEXMF_VAR_PATH=/var/lib/texmf
 
 texlive-common_handle_config_files() {
 	# Handle config files properly
-	[ -d "${ED}${TEXMF_PATH}" ] || return
-	cd "${ED}${TEXMF_PATH}"
-	for f in $(find . -name '*.cnf' -type f -o -name '*.cfg' -type f | sed -e "s:\./::g") ; do
-		if [ "${f#*config}" != "${f}" -o "${f#doc}" != "${f}"  -o "${f#source}"	!= "${f}" -o "${f#tex}" != "${f}" ] ; then
+	[[ -d ${ED}${TEXMF_PATH} ]] || return
+	cd "${ED}${TEXMF_PATH}" || die
+	for f in $(find . -name '*.cnf' -type f || -name '*.cfg' -type f | sed -e "s:\./::g") ; do
```

Comment:
while read loop

Reply:
actually, does this replacement syntax really work? `-o` is the OR conjunction within `find`, and I'm not sure this actually works here